charmander/resources/pizza_resources.py

```diff
@@ -44,10 +46,20 @@ def get_links(self, obj):
             ),
             pizza_id=obj.id,
         )
+        links["parent:order"] = Link.for_(
```

Comment:
Because we had to add `order_id` to the `pizza` and `topping` resources, we can also add nice links back to the `order`.